src/ert/job_queue/driver.py

```diff
@@ -101,65 +106,130 @@ async def kill(self, realization: "RealizationState") -> None:
         realization.verify_kill()
 
 
+LSF_STATUSES = [
```

Comment:
This does not need to a global list available to all drivers, it can move the LSFDriver class